tools/distribution/src/utils.py

```diff
@@ -71,9 +71,11 @@ def read_sdk_version() -> str:
     Reads SDK version from 'Sources/Datadog/Versioning.swift'.
     """
     file = 'DatadogCore/Sources/Versioning.swift'
-    version_regex = r'^.+\"([0-9]+\.[0-9]+\.[0-9]+[\-a-z0-9]*)\"'  # e.g. 'internal let __sdkVersion = "1.8.0-beta1"'
```

Comment:
This regex is no longer valid following 856ed98b034713fc340452ff8d26a2ebc3834dec
It has been replace with [VERSION_PATTERN](https://packaging.pypa.io/en/latest/version.html#packaging.version.VERSION_PATTERN)